index.html

```diff
@@ -88,18 +93,21 @@
     <script src="ol3dc/js/interpolator/interpolator.js"></script>
     <script src="ol3dc/js/entry/entry-manager.js"></script>
     <script src="ol3dc/js/effect/blur.js"></script>
+    <script src="ol3dc/js/effect/backgroundblur.js"></script>
     <script src="ol3dc/js/effect/frontlight.js"></script>
     <script src="ol3dc/js/effect/toplight.js"></script>
     <script src="ol3dc/js/effect/backlight.js"></script>
     <script src="ol3dc/js/effect/rain.js"></script>
     <script src="ol3dc/js/effect/effect-manager.js"></script>
     <script src="ol3dc/js/config/config-manager.js"></script>
     <script src="ol3dc/js/gui/multi-language.js"></script>
-    <script src="ol3dc/js/gui/gui-layout.js"></script>
     <script src="ol3dc/js/vrm/vrm-manager.js"></script>
     <script src="ol3dc/js/control/health-manager.js"></script>
     <script src="ol3dc/js/control/control-manager.js"></script>
     <script src="ol3dc/js/control/hotkey.js"></script>
+    <script src="ol3dc/js/gui/gui-layout.js"></script>
+    <script src="ol3dc/js/clientStorage/index.js"></script>
```

Comment:
Let's move this import higher up, since this is independent from others.

Reply:
I will do that :)